_includes/quick-guide-table.html

```diff
@@ -0,0 +1,121 @@
+  font-weight: 400;
+  border-radius: 22px;
+  font-stretch: normal;
+  font-size: 1rem;
+  justify-content: center;
+  align-items: center;
+  line-height: 1.7;
+  text-align: justify;
+}
+.flex-container .push {
+    width: 200px;
+    text-align: center;
+    margin-left: auto;
+    margin-right: auto;
+    border-radius: 22px;
+    padding-top: 2rem;
+}
+.arrow{
+    width: 0px;
+    height: 10px;
+    border:30px solid transparent;
+    margin:auto;
+    margin-top: 20px;
+  }
+
+  .arrow.arrow-down{
+    border-top-color: rgba(102, 89, 89, 0.637);
+    border-bottom-width: 0px;
+    align-self: center;
+  }
+  
+
+  </style>
+
+<!-- <table>
+    <colgroup>
+    <col width="30%" />
+    <col width="70%" />
+    </colgroup>
+    <thead>
+    <tr class="header">
+    <th>Steps</th>
+    <th>Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td markdown="span">Checklist Requirement</td>
+        <td markdown="span"> Defining the information about the type of data that can be submitted.</td>
+    </tr>
+    <tr>
+    <td markdown="span">Get your submission account</td>
+    <td markdown="span">Fill the submission form and provide details of the data type(s), used platform(s) and estimated size of your submission. 
+        If you are affiliated to an existing consortium, such as the International Cancer Genome Project (ICGC), please add this information in the comments.</td>
+    </tr>
+    <tr>
+    <td markdown="span">Register Study/DAC</td>
+    <td markdown="span"> Use Submitter Portal to register your study, samples, Data Access Committee (DAC) and Policy.</td>
+    </tr>
+    <tr>
+        <td markdown="span">Upload data</td>
+        <td markdown="span"> Encrypt your data files using the EgaCryptor or encrypt your files localy and upload it using default FTP clients or Aspera.</td>
+    </tr>
+    <tr>
+        <td markdown="span">Register experiments and runs</td>
+        <td markdown="span"> Associate each data file to a registered sample and study by Linking files to samples . Details of the experimental procedure you followed must be provided.</td>
+    </tr>
+    <tr>
+        <td markdown="span">Finish your submission</td>
+        <td markdown="span"> Group your runs/analysis into datasets and link them to your new or existing DAC and policy using Submitter Portal 
+        or an XML based programmatic submission. Data request are done at a dataset level, thus files within a datasets must share release conditions.</td>
+    </tr>
+    <tr>
+        <td markdown="span">Release and admin</td>
+        <td markdown="span"> Instruct our Helpdesk to release your study. 
+        All registered studies are automatically placed on hold until the named submission or DAC contact instructs our Helpdesk for the study to be released.</td>
+    </tr>
+
+    </tbody>
+</table> -->
+
+
+<div class = "flex-container">
+    <div class="push">1. Checklist Requirements <div class="arrow arrow-down"></div> </div> 
```

Comment:
I feel that "checklist requirements" is not so easy to understand. Can we write "Is SDA the right archive?" instead?

Reply:
Done!